src/devices/controller.c

```diff
@@ -259,8 +148,16 @@ int32_t controller_print(controller_id_e_t id, uint8_t line, uint8_t col, const
 }
 
 int32_t controller_clear_line(controller_id_e_t id, uint8_t line) {
-	static const char* clear = "               ";
-	return controller_print(id, line, 0, clear);
+	uint8_t port;
+	CONTROLLER_PORT_MUTEX_TAKE(id,port);
+	static char* clear = "";
+	uint32_t rtn_val = vexControllerTextSet(id, ++line, 1, clear);
+	internal_port_mutex_give(port);
```

Comment:
Ideally we would switch on the vexSystemVersion like we do for controller_clear.